tasks/issue.py

```diff
@@ -40,3 +40,57 @@ def assign_owner(_, issue_id, dry_run=False):
 @task
 def generate_model(_):
     fetch_data_and_train_model()
+
+
+WAVES = [
+    "wave",
+    "waveboi",
+    "wastelands-wave",
+    "wave_hello",
+    "wave-hokusai",
+    "wave_moomin",
+    "wave2",
+    "wave3",
+    "wallee-wave",
+    "vaporeon_wave",
+    "turtle-wave",
+    "softwave",
+    "shiba-wave",
+    "minion-wave",
+    "meow_wave_comfy",
+    "mario-wave",
+    "link-wave",
+    "kirby_wave",
+    "frog-wave",
+    "fox_wave",
+    "duckwave",
+    "cyr-wave",
+    "cozy-wave",
+    "cat-wave",
+    "capy-wave",
+    "bufo-wave",
+    "bongo-wave",
+    "blobwave",
+    "birb-wave",
+    "arnaud-wave",
+]
+
+
+@task
+def ask_reviews(_, pr_id):
+    gh = GithubAPI()
+    pr = gh.repo.get_pull(int(pr_id))
+    if any(label.name == 'ask-review' for label in pr.get_labels()):
+        reviewers = [f"@datadog/{team.slug}" for team in pr.requested_teams]
+        import random
+
+        from slack_sdk import WebClient
+
+        client = WebClient(os.environ['SLACK_API_TOKEN'])
+        for reviewer in reviewers:
+            channel = next(
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Should we print a different message when we don't find a channel? It would help looking for owners and asking to update the map

Reply:
Nice suggestion, I'll take care of this, thanks